.travis.yml

```diff
@@ -22,7 +22,14 @@ env:
      - "TESTDIR=C++/treefrog"
      - "TESTDIR=C++/ulib"
      - "TESTDIR=C++/wt"
-     - "TESTLANG=Clojure/immutant"
+     - "TESTDIR=Clojure/aleph"
+     - "TESTDIR=Clojure/compojure"
+     - "TESTDIR=Clojure/http-kit"
+     - "TESTDIR=Clojure/immutant"
+     - "TESTDIR=Clojure/luminus"
+     - "TESTDIR=Clojure/macchiato"
+     - "TESTDIR=Clojure/pedestal"
+     - "TESTDIR=Clojure/reitit"
```

Comment:
I see what happened, please remove all these and just add `- "TESTLANG=Clojure"`, that will test all the Clojure implementations.